<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableView</class>
 <widget class="QWidget" name="TableView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>787</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>表格视图</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_data_filter">
       <property name="text">
        <string>数据过滤</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_data_filter">
       <property name="text">
        <string>/*</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>表格视图</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
